packages/vuetify/src/components/VFileInput/__tests__/VFileInput.spec.cy.tsx

```diff
@@ -136,4 +136,48 @@ describe('VFileInput', () => {
       .get('.v-file-input input')
       .should('have.attr', 'accept', 'image/*')
   })
+
+  it.only('should handle the same file to be selected via controlled model', () => {
```

Comment:
focusing until we work out how this test should look like.

Reply:
try using `attachFile()`